packages/worker/src/runtime.ts

```diff
@@ -137,51 +121,38 @@ export class Core {
    * a new Core instance will be installed and configured to connect to
    * a local server.
    */
-  public static async instance<T extends typeof Core>(this: T): Promise<InstanceType<T>> {
```

Comment:
Considering making this async in case init will become async in the future.
Another option would be to remove log forwarding altogether and moving the logger into the Worker.
Not 100% sure what to do with this.
Could be in the next PR though but should be addressed before we make a release.

Reply:
Anything that would require calling into the bridge loop would make this async.
Changing it later from `sync` to `async` will be painful for users.